clues: fix incorrect clue itemIds

TheLope opened the pull request:
1. Item ID should be 7276
    - https://oldschool.runescape.wiki/w/Clue_scroll_(medium)_-_GOT_A_BOY
2. Item ID should be 3493
    - https://oldschool.runescape.wiki/w/Clue_scroll_(easy)_-_Search_the_chests_in_the_Dwarven_Mine

Nightfirecat commented:
I've added a commit to test all the clue types with IDs defined for duplicates. This identified one more place where duplicate IDs are present, namely for the cryptic clues with the following hints:
* Search the chests in the Dwarven Mine.
* Search through some drawers in the upstairs of a house in Rimmington.

From personal experience, I believe the ID given is for the latter clue (the Rimmington drawers) however I recommend seeing that it is tested before committing to that change. Once the tests pass, this fix is good to go.

TheLope commented:
> I've added a commit to test all the clue types with IDs defined for duplicates. This identified one more place where duplicate IDs are present, namely for the cryptic clues with the following hints:
> 
> * Search the chests in the Dwarven Mine.
> * Search through some drawers in the upstairs of a house in Rimmington.
> 
> From personal experience, I believe the ID given is for the latter clue (the Rimmington drawers) however I recommend seeing that it is tested before committing to that change. Once the tests pass, this fix is good to go.

Confirmed and updated. Tests passing.
